bin/commands/init/apfauth/index.sh

```diff
@@ -11,55 +11,14 @@
 # Copyright Contributors to the Zowe Project.
 #######################################################################
 
-print_level1_message "APF authorize load libraries"
+USE_CONFIGMGR=$(check_configmgr_enabled)
+if [ "${USE_CONFIGMGR}" = "true" ]; then
+  if [ -z "${ZWE_PRIVATE_TMP_MERGED_YAML_DIR}" ]; then
 
-###############################
-# constants
-auth_libs="authLoadlib authPluginLib"
-
-###############################
-# validation
-require_zowe_yaml
-
-# read prefix and validate
-prefix=$(read_yaml "${ZWE_CLI_PARAMETER_CONFIG}" ".zowe.setup.dataset.prefix")
-if [ -z "${prefix}" ]; then
-  print_error_and_exit "Error ZWEL0157E: Zowe dataset prefix (zowe.setup.dataset.prefix) is not defined in Zowe YAML configuration file." "" 157
-fi
-
-###############################
-# APF authorize loadlib
-job_has_failures=
-for key in ${auth_libs}; do
-  # read def and validate
-  ds=$(read_yaml "${ZWE_CLI_PARAMETER_CONFIG}" ".zowe.setup.dataset.${key}")
-  if [ -z "${ds}" ]; then
-    # authLoadlib can be empty
-    if [ "${key}" = "authLoadlib" ]; then
-      ds="${prefix}.${ZWE_PRIVATE_DS_SZWEAUTH}"
-    else
-      print_error_and_exit "Error ZWEL0157E: ${name} (zowe.setup.dataset.${key}) is not defined in Zowe YAML configuration file." "" 157
-    fi
+    # user-facing command, use tmpdir to not mess up workspace permissions
+    export ZWE_PRIVATE_TMP_MERGED_YAML_DIR=1
   fi
-
-  print_message "APF authorize ${ds}"
-  apf_authorize_data_set "${ds}"
-  code=$?
-  if [ $code -ne 0 ]; then
-    if [ "${ZWE_CLI_PARAMETER_IGNORE_SECURITY_FAILURES}" = "true" ]; then
-      job_has_failures=true
-    else
-      exit $code
-    fi
-  else
-    print_debug "- APF authorized successfully."
-  fi
-done
-
-###############################
-# exit message
-if [ "${job_has_failures}" = "true" ]; then
-  print_level2_message "Failed to APF authorize Zowe load libraries. Please check log for details."
+  _CEE_RUNOPTS="XPLINK(ON),HEAPPOOLS(OFF),HEAPPOOLS64(OFF)" ${ZWE_zowe_runtimeDirectory}/bin/utils/configmgr -script "${ZWE_zowe_runtimeDirectory}/bin/commands/init/apfauth/cli.js"
 else
-  print_level2_message "Zowe load libraries are APF authorized successfully."
+  print_error_and_exit "Error ZWEL0316E: Command requires zowe.useConfigmgr=true to use." "" 316
```

Comment:
The function `print_error_and_exit` is using `exit $code` as the last statement. The `exit` command takes the `$code` and treats it as `byte` type. So this line will return **60** (316-256). Same for other commands and any error number above 256.

Reply:
I just removed it because I was thinking this is a harsh and pointless error - instead of warning about setting `zowe.useConfigmgr=true`, since this code is configmgr-only now, just dont even read the variable...